stubs/Pillow/PIL/PdfParser.pyi

```diff
@@ -47,10 +47,10 @@ class PdfName:
 class PdfArray(list[Any]):
     def __bytes__(self): ...
 
-class PdfDict(collections.UserDict):
+class PdfDict(collections.UserDict[Any, Any]):
```

Comment:
Maybe:

```suggestion
class PdfDict(collections.UserDict[bytes, Any]):
```

Reply:
I was very confused about this class, possibly because I haven't worked with `UserDict` before and sub-classing `dict` works differently. But can't the Key either be a `bytes` or the literal string `'data'`?

Reply:
With the string `"data"`, it calls `UserDict.__setattr__` which sets `self.data` to the value. It has nothing to do with `self[...]`.

Reply:
Still confused, but I'll do what you say. :)

Reply:
I mixed up `__setitem__` and `__setattr__`, makes sense now.